content/pages/venir.rst

```diff
@@ -49,17 +43,38 @@ consulter `le site du Réseau Bus+Métro de Rennes`_.
 .. _`le site du Réseau Bus+Métro de Rennes`: http://www.star.fr/se-deplacer/fiches-horaires-et-plans/?tx_pnfstarod_searchdocument%5Baction%5D=search&tx_pnfstarod_searchdocument%5Bcontroller%5D=SearchLines&cHash=2bdb192917b9dd2b387895b85238c8ed
 
 
-En train
-========
+Vélo libre service
+++++++++++++++++++++++
+
+Rennes dispose d'un système de vélos en libre service complémentaire des transports en commun : `vélostar <https://www.levelostar.fr/fr/>`_
+
+Venir à Rennes en voiture
+============================
+
+Si vous souhaitez venir en voiture, nous vous invitons à partager votre trajet
+avec d'autres personnes. Nous mettons à votre disposition `un espace de
+covoiturage <https://pad.notmyidea.org/p/covoiturage-pyconfr>`_ ou vous pouvez indiquer votre trajet et vos contacts, si vous le
+souhaitez.
+
+Vous pouvez `calculer votre itinéraire avec Open Street Map <http://www.openstreetmap.org/directions#map=9/47.9531/-1.8196>`_
+
+
+Venir à Rennes en train
+==========================
```

Comment:
It actually has to be equal or greater than the text length, so it should work (but indeed it's better to have the same size) 

Le 14 juin 2016 13:37:25 GMT+02:00, "Rémy HUBSCHER" <notifications@github.com> a écrit :
>> +
>> +Rennes dispose d'un système de vélos en libre service complémentaire
>des transports en commun : `vélostar <https://www.levelostar.fr/fr/>`_
>> +
>> +Venir à Rennes en voiture
>> +============================
>> +
>> +Si vous souhaitez venir en voiture, nous vous invitons à partager
>votre trajet
>> +avec d'autres personnes. Nous mettons à votre disposition `un espace
>de
>> +covoiturage <https://pad.notmyidea.org/p/covoiturage-pyconfr>`_ ou
>vous pouvez indiquer votre trajet et vos contacts, si vous le
>> +souhaitez.
>> +
>> +Vous pouvez `calculer votre itinéraire avec Open Street Map
><http://www.openstreetmap.org/directions#map=9/47.9531/-1.8196>`_
>> +
>> +
>> +Venir à Rennes en train
>> +==========================
>
>The size is not good, it should match the text length
>
>---
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/AFPy/pyconfr_2016/pull/13/files/3b96cd44209af5c2f29dd2105d8704eaccd890de#r66953106